profiles-sdk-examples/README.md

```diff
@@ -0,0 +1,191 @@
+* The output of the skill is the Job execution logs
+* The [docker image](./main-app/src/main/resources/Dockerfile) for the `main-app` uses
+  a [Spark Submit](https://spark.apache.org/docs/latest/submitting-applications.html) based wrapper to run the Spark
+  application. The resources for the spark-submit wrapper is in the [main-app/src/main/resources/python/](./main-app/src/main/resources/python)
+  directory and is necessary for packaging as a Skill.
+
+**Before creating the skill**, you will need to:
+* Update the [spark-conf.json](./main-app/src/main/resources/conf/spark-conf.json) with the CLI application command and other config options.
+* Verify skills `payload.json` file refers to the above Spark configuration file (in the built container).
+
+To build the skill, you will need to set the following environment variables:
+* `DOCKER_PREGISTRY_URL=xxxx` - Private Registry Url accessible from Cortex
+* `PROJECT_NAME=xxxx` - Name of the project to save the skill, action and types
+* `CORTEX_TOKEN=xxxx` - For connecting to Cortex
+
+Building the Skill
+```
+make clean build create-app-image deploy-skill
+...
+
+latest: digest: sha256:d90eb1af0789bfa9b36082d9ef67ffc535e2f2a3f8c62c10950dff14a5fd57dc size: 2606
+cortex types save -y templates/types.yaml --project testi-69257
+Type definition saved
+cortex actions deploy --actionName profiles-example --actionType job --docker private-registry.dci-dev.dev-eks.insights.ai/profiles-example:latest --project testi-69257 --cmd '["scuttle", "python", "submit_job.py"]' --podspec ./templates/podspec.yaml
+{
+  "success": true,
+  "action": {
+    "_isDeleted": false,
+    "_projectId": "testi-69257",
+    "_createdBy": "laguirre@cognitivescale.com",
+    "name": "profiles-example",
+    "description": "",
+    "image": "private-registry.dci-dev.dev-eks.insights.ai/profiles-example:latest",
+    "type": "job",
+    "command": [
+      "scuttle",
+      "python",
+      "submit_job.py"
+    ],
+    "scaleCount": 1,
+    "podSpec": "[{\"path\":\"/containers/0/imagePullPolicy\",\"value\":\"Always\"}]",
+    "jobTimeout": 0,
+    "k8sResources": [],
+    "environmentVariables": null,
+    "createdAt": "2022-07-07T17:10:14.258Z",
+    "updatedAt": "2022-07-07T17:10:14.258Z",
+    "_version": 2
+  }
+}
+cortex skills save -y templates/skill.yaml --project testi-69257
+Skill saved: {"success":true,"version":2,"message":"Skill definition profiles-example saved."}
+```
+
+Invoking the Skill:
+```
+$ make invoke
+cortex skills invoke --params-file templates/payload.json profiles-example params --project laguirre-testi-69257
+{
+  "success": true,
+  "activationId": "115d1196-408a-47fa-91c4-6f8e8a391641"
+}
+```
+
+Run `cortex agents get-activation <activation-id>` or `cortex tasks logs <task-name>` to view logs of the skill activation.
+
+## FAQ
```

Comment:
Fine by me. Will delete. I threw these questions in here because I figured there would be eventually be an FAQ style section (and these wanted to have this written somewhere)